packages/vitest/src/index.ts

```diff
@@ -84,8 +84,8 @@ declare global {
       toContainEqual<E>(item: E): void
       toBeTruthy(): void
       toBeFalsy(): void
-      toBeGreaterThan(num: number): void
-      toBeGreaterThanOrEqual(num: number): void
+      toBeGreaterThan(num: number | bigint): void
+      toBeGreaterThanOrEqual(num: number | bigint): void
       toBeLessThan(num: number): void
       toBeLessThanOrEqual(num: number): void
```

Comment:
Great PR! `toBeLessThan` and `toBeLessThanOrEqual` should also be `number | bigint`. Would you like to modify them as part of this PR?